<launch>
	<param name="root_dir" value="$(find living_lab_robot_perception)"/>
	<node name="u2net" type="u2net_demo.py" pkg="living_lab_robot_perception" output="screen" respawn="true"/>
</launch>
